### PR TITLE
adds a button to copy the generated config to the buffer as base64 string

### PR DESCRIPTION
**Pull request by trin4ik**
string like: 
$ echo 'base64string' | base64 --decode > nginxconfig.io.zip
nginxconfig.io

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 [![nginxconfig.io](public/assets/img/github.png)](https://nginxconfig.io)
 
+Fork adds a button to copy the generated config to the buffer as base64 string
```
Comment (just-boris) [inline README.md]: This was supposed to be displayed in the fork, but I do not think it is needed for the main repo.

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
 		"bootstrap": "^4.2.1",
 		"eslint": "^5.9.0",
 		"highlight.js": "^9.12.0",
+		"js-base64": "^2.5.1",
```
Comment (just-boris) [inline package.json]: Why do you need this dependency? There are no occurrences of it below.

File: public/index.html [hunk 3]
```diff
@@ -998,6 +998,13 @@
 					</div>
 					<div class="grid-item col-xl-12 mb-3 text-center" ng-cloak>
 						<button class="btn btn-primary btn-sm btn-download" ng-click="downloadZip()"><img src="assets/img/download.svg" alt="Download"> Generated config <small>(.zip)</small></button>
+						<button tooltips tooltip-template="Copied!" tooltip-show-trigger="click" class="btn btn-success btn-sm btn-download btn-base64" ng-click="downloadBase64()"><img src="assets/img/download.svg" alt="Download"> Copy Base64 <small>(inline)</small></button>
+					</div>
+					<div id="base64-zip-wrapper" class="custom-control-input">
+						<button class="btn btn-light btn-clipboard" id="btn-base64-zip-line" ngclipboard data-clipboard-target="#base64-zip-line" ngclipboard-success="clipboardSuccess('base64-zip-line')">
+							<img src="assets/img/clipboard-dark.svg" alt="Copy to clipboard">
+						</button>
+						<div id="base64-zip-line"></div>
```
Comment (just-boris) [inline public/index.html]: There are missing styles to make this block with the text invisible

File: README.md (hunk 1 above)
Comment (trin4ik) [reply README.md]: true

File: package.json (hunk 2 above)
Comment (trin4ik) [reply package.json]: only 4 cp to assets, mb later all js assets will be generated by webpack. but now its redundantly, yes

File: public/index.html (hunk 3 above)
Comment (trin4ik) [reply public/index.html]: in parent item i have class custom-control-input, this is opacity, position and z-index styles
```suggestion
					<div id="base64-zip-wrapper" class="custom-control-input">
```

Comment (trin4ik) [reply public/index.html]: in parent item i have class custom-control-input, this is opacity, position and z-index styles

**Comment by valentinxxx** [comment]
Thanks for the PR! 😉

**Pull request merged**